docs/source/model_fitting.rst

```diff
@@ -4,7 +4,7 @@ Model Fitting
 =============
 
 SpecViz utilizes
-`Astropy Models and Fitting <http://astropy.readthedocs.org/en/latest/modeling/index.html>`_
+`Astropy Models and Fitting <https://astropy.readthedocs.io/en/latest/modeling/index.html>`_
```

Comment:
Thanks for doing this! While you are at this, would you mind changing this to use intersphinx instead? That is, link it like this:
```
:ref:`Astropy Models and Fitting <astropy:astropy-modeling>`
```